rust/worker/src/system/wrapped_message.rs

```diff
@@ -1,6 +1,30 @@
 use super::{Component, ComponentContext, Handler};
 use async_trait::async_trait;
 use std::fmt::Debug;
+use tokio::sync::oneshot;
+
+// Why is this separate from the WrappedMessage struct? The size of WrappedMessage
```

Comment:
"The size of WrappedMessage
// must be constant for all messages on a given component"

I don't follow this? Is this new / why?

Reply:
my reasoning may be faulty, I'll go through this again